lib/PostMessageTransport.js

```diff
@@ -58,6 +61,15 @@ define(function (require, exports, module) {
     function send(idOrArray, msgStr){
         var win = _iframeRef.contentWindow;
 
+        // Parse msgStr and check to see if it's a reload command.
+        // If so, trigger launcher.launch and return
+        var msg = JSON.parse(msgStr);
+
+        if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
```

Comment:
Yes, so we shouldn't have it reload if it's a CSS file. We could also have it reload the HTML that it's linked to, if applicable?